timur/lib/client/jsx/utils/attributes.js

```diff
@@ -0,0 +1,17 @@
+const order = ({attribute_type}) =>
+  ({
+    parent: 1,
+    identifier: 2,
+    collection: 3,
+    table: 4,
+    child: 5,
+    link: 6,
+    file: 7,
+    image: 8
+  }[attribute_type] || 9);
+
+export const sortAttributes = (attributes) =>
```

Comment:
Could we use this function [here](https://github.com/mountetna/monoetna/blob/master/timur/lib/client/jsx/selectors/search.js#L78) as well, for even mooore consistency?